src/applications/686c-674/config/chapters/report-dependent-death/helpers.js

```diff
@@ -1,8 +1,8 @@
-export const relationshipEnums = ['SPOUSE', 'DEPENDENT_PARENT', 'CHILD'];
+export const relationshipEnums = ['spouse', 'parent', 'child'];
 export const relationshipLabels = {
-  SPOUSE: 'Spouse',
```

Comment:
One of the unit test failures seemed to be because the `relationshipLabels.spouse` keyname was changed to `relationshipLabels.SPOUSE` without updating the test.